content/en/tracing/trace_collection/proxy_setup/_index.md

```diff
@@ -657,6 +661,70 @@ More configuration options can be found on the [kong-plugin-ddtrace][3] plugin d
 [3]: https://github.com/DataDog/kong-plugin-ddtrace#configuration
 
 {{% /tab %}}
+
+{{% tab "Apache HTTP Server" %}}
+
+Datadog provides an HTTPd [module][1] to enhance [Apache HTTP Server][2] and [IHS HTTP Server][3] capabilities with APM Tracing.
+
+### Compatibility
+
+- The module only supports Apache HTTP Server `v2.4.x`.
+- Since IHS HTTP Server is essentially a wrapper of the Appache HTTP Server, the module can also be used with IHS without any modifications.
+
+### Installation
+
+<div class="alert alert-warning">
+  <strong>Note</strong>: Only Apache HTTP Server 2.4.x for x86_64 architecture is supported.
+</div>
```

Comment:
```suggestion
```
Is there a reason you want to mention this twice (since it's also in the compatibility section)? If not, I think we should remove it.

Reply:
Hm... not intended. We should keep the one in the `<div>` and remove the one in the compatibility section. good catch @brett0000FF

Reply:
@dmehala - Great, sounds good! I just made that change for you. This looks good to me now. Just wanted to confirm, is this ready to merge?

Reply:
@brett0000FF Thank you for updating the doc. This is ready to merge. 🙏